massgov/mayflower
# DP-6198: moving work from broken branch to new

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description
Updates tabbing order and sets keyboard open/close for utility menu

## Related Issue / Ticket
https://jira.mass.gov/browse/DP-6198

## Steps to Test
From any page with a header, tab around in the navigation. 

1. The utility nav options are now reachable by tab, opening and closing as expected
2. Tab through the main nav items, they tabbing now catches all items
3. From a menu item, click enter, spacebar, or an up or down arrow to see the submenu open
4. Navigate through the submenu using tabs or arrows. Enter selects the link


## Screenshots
![menu](https://user-images.githubusercontent.com/18662734/53374104-fee81180-391c-11e9-9f87-81b15b3334bf.gif)
